src/xdocs-examples/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/innerassignment/Example2.java

```diff
@@ -0,0 +1,47 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="InnerAssignment"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.innerassignment;
+
+//non-compiled with javac: Compilable with Java14
```

Comment:
@romani @nrmancuso Do we check that these files are compilable with different java versions? I don't remember seeing this before with examples.
Shouldn't this also be before the package?

Reply:
I put it before the package

Reply:
@rnveach We only compile with the specified version.